packages/teams-js/test/utils.ts

```diff
@@ -39,10 +39,10 @@ export class Utils {
 
     this.parentWindow = {
       postMessage: function(message: MessageRequest, targetOrigin: string): void {
-        if (message.func === 'initialize') {
-          expect(targetOrigin).toEqual('*');
-        } else {
-          expect(targetOrigin).toEqual(that.validOrigin);
```

Comment:
I really don't think this is a good pattern to use. Putting expect checks in shared functions used by all tests means that we end up with a bunch of hidden expects that people don't know about and make it very hard to use expect.assertions() anywhere.

Reply:
Agree!